### DISCUSSION TITLE
[Feature Request] Need support for ppc64le architecture

### DISCUSSION THREAD

**Discussion by amintek** [grafana/grafana#39583] [General]
hi guys, i have a Kubernetes cluster deployed on Linux/ppc64le platform, and i'm trying to figure out how to install grafana to monitor my k8s cluster since i didn't found any ppc64le image in grafana/grafana docker hub. thanks in advance

**Comment by MatthieuSarter** [comment]
Hi,

I support this feature request, I was going to ask for it :)

It would be great to have an official ppc64le images, this would help using Grafana and other projects depending on it (like istio).

FYI I have been able to build a ppc64le image from the main branch, without any change, and the test results are pretty good, I'd say far less than 1% of the test cases failing (I can provide more precise details if needed).

**Comment by zorkin82** [comment]
I would also be happy about a official support!
Non the less there is a available community build at: https://www.power-devops.com/grafana

**Comment by clnperez** [comment]
This is a good point @zorkin82. We (IBM Power and some collaborators like Unicamp) have been doing on-premise builds of grafana and other projects for customers, so we've probably hurt ourselves a little by not funnelling customers' requests back to the communities. It used to be more difficult to get other architectures' binaries and containers built, but now that projects have other tools available (easy cross-building, Power nodes available in a few CI platforms, etc.) -- we'd love to rethink this strategy where we can!

**Comment by clnperez** [comment]
@amintek can you update the title of this to indicate that it's a feature request? we were asked to create one a while ago: https://github.com/grafana/grafana/pull/23177

**Comment by eric-aquaronne** [comment]
Hello, I see demand for Grafana on PPC64, and with leverage of the CPU accelerators (4 per core), so the right libraries, can this be made to happen OR do we have to just : find an alternative (RH OCP functions, do our own build) ?????
Many IBM Power users are moving to Linux on Power (better MTBF, cheaper compute by Watts....) : we need the support

**Comment by lehrig** [comment]
I second that, I have also seen demand for Grafana Enterprise on ppc64le...

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82976. Feel free to continue the discussion around this there. Thank you!

**Comment by NicolaeCh** [comment]
Just published a container for ppc64le: https://hub.docker.com/r/nchirea/grafana-ppc64le
